mobile/src/main/java/com/gelakinetic/mtgfam/FamiliarActivity.java

```diff
@@ -681,7 +681,7 @@ public void onDrawerSlide(View drawerView, float slideOffset) {
                 }
             }
         };
-        mDrawerLayout.setDrawerListener(mDrawerToggle);
+        mDrawerLayout.addDrawerListener(mDrawerToggle);
```

Comment:
Yes. Android Studio 3 (currently in canary), tells you anything that is depreciated and you can use the updated function for it (as opposed to before it told you all depreciations regardless of if they were relevant or not). Tested and working on API 15.